test/unified_format.py

```diff
@@ -1770,15 +1770,16 @@ def check_log_messages(self, operations, spec):
         def format_logs(log_list):
             client_to_log = defaultdict(list)
             for log in log_list:
-                data = json_util.loads(log.message)
-                client = data.pop("clientId")
-                client_to_log[client].append(
-                    {
-                        "level": log.levelname.lower(),
-                        "component": log.name.replace("pymongo.", "", 1),
-                        "data": data,
-                    }
-                )
+                if log.module != "ocsp_support":
```

Comment:
Suggest writing like this to avoid code churn:
```python
if log.module == "ocsp_support":
    continue
```

Reply:
Eh too late I guess.

Reply:
I can open another, I agree your suggestions make for cleaner code.